package.json

```diff
@@ -2,15 +2,15 @@
     "name": "@guardian/consent-management-platform",
     "version": "3.0.4",
     "description": "Library of useful utilities for managing consent state across *.theguardian.com",
-    "main": "lib/index.js",
-    "types": "lib/index.d.ts",
+    "main": "dist/index.js",
+    "module": "dist/index.esm.js",
+    "types": "dist/index.d.ts",
     "scripts": {
-        "build": "yarn validate && yarn build-only",
-        "build-only": "yarn clean && rollup --config && yarn declaration",
-        "clean": "rm -rf lib/*",
+        "build": "yarn clean && rollup --config && yarn build:types",
+        "clean": "rm -rf dist/*",
         "cy": "cypress open",
         "cy:run": "cypress run",
-        "declaration": "tsc --emitDeclarationOnly -d -m commonjs ",
+        "build:types": "tsc --declaration --emitDeclarationOnly --declarationDir 'dist'",
```

Comment:
Non-blocking comment: The simplification of the library (not exposing a React component) in combination with the changes to the build process could mean we can go back to having the declaration flag in the tsconfig file and getting rid of this separate step to build types.